sw/device/lib/dif/dif_otbn.c

```diff
@@ -213,33 +232,17 @@ dif_otbn_result_t dif_otbn_is_busy(const dif_otbn_t *otbn, bool *busy) {
   return kDifOtbnOk;
 }
 
-dif_otbn_result_t dif_otbn_get_err_code(const dif_otbn_t *otbn,
```

Comment:
I'd suggest to keep this function name dif_otbn_get_err_code(). No matter how we call the signal internally, the register is called ERR_CODE, and if I'd be searching for a function that gets me the information from this register, I'd expect the function to have `ERR_CODE` in its name as well.

Reply:
In that case I think I'd prefer just naming the register ERR_BITS. As I said I think calling a bitfield that can have bits set independently a code could cause confusion.